workspace_host/interface.js

```diff
@@ -207,6 +207,7 @@ async.series([
         });
         watcher.on('add', filename => {
             // Handle new files
+            logger.info(`Watching file ${filename}`);
```

Comment:
Should we also log `change` events? Those are the ones I've been testing by editing a file.